### ISSUE TITLE
Cannot press the alt key

### ISSUE THREAD

**Issue by reteps** [utmapp/UTM#3413]
**Describe the issue**

I am trying to run windows server 2019 for a course I am taking on udemy. To unlock the machine, one must press "control + alt + delete". I cannot figure out how to send this key combination to the machine and unlock it for use.

![image](https://user-images.githubusercontent.com/13869303/147419241-9d8e619f-071b-47aa-8954-61457245f23d.png)

I have tried the combinations: "control + option + delete" and "command + option + delete" but neither one seems to work.

**Configuration**
* UTM Version: 2.4.1 (40)
* OS Version: Big Sur, 11.5.1
* M1

**Comment by reteps** [comment]
Solution was to plug in a standard keyboard, then disable secure login. https://answers.microsoft.com/en-us/windows/forum/all/disable-ctrlaltdel-login-in-windows-10-home/cbc90186-114b-4720-a3aa-ba30c7b911f5?auth=1

**Comment by conath** [comment]
You probably need to press fn + ⌃ + ⌥ + delete because Windows expects a "forward delete" (delete next character) where Macs use the word "delete" to represent the Windows key "backspace" (delete previous character).

**Comment by molehauling** [comment]
does not work

**Comment by bdbasinger** [comment]
"Control + Option + Fn + Delete" should work

**Comment by syanyong** [comment]
It works for me on UTM 3.2.4. Host OS Monterey 12.5. Guest OS: Windows Server 2012

**Comment by RobLW** [comment]
On MacBook pro M1 2021, Venture 13.1 and UTM 4.0.9. FN+Ctrl+Opt+backspace worked a charm. Many thanks.

**Comment by BantuSpirit** [comment]
> "Control + Option + Fn + Delete" should work

Thanks @bdbasinger this option worked for me

**Comment by jakerobb** [comment]
FWIW, the above solution didn't work for me.

I have an external extended Apple keyboard with the TouchID button, so I have a regular forward-delete button.

Environment:
M3 Pro Macbook Pro
Sonoma 14.5
UTM 4.5.2 (97)
Installing Windows 10 Pro on a brand new instance, using an ISO downloaded via CrystalFetch
Trying to control-alt-delete during the OOBE process

I have tried:
* control-alt-delete using the external keyboard
* control-alt-fn-delete using the external keyboard (didn't think this would work; just being thorough)
* control-alt-fn-backspace using the external keyboard
* control-alt-fn-backspace using the built-in keyboard
* I've tried all of the above using the right-side control and option keys instead where applicable
* changing UTM settings so that it uses command-option instead of control-option for input capture, to be sure it's not interfering, then trying all of the above again

Keyboard input in general works fine.

This isn't blocking me or anything, so at the moment I'm content to just comment here. 

I really think #5684 would be great.
